### ISSUE TITLE
Missing TimeLineRegionEnter event

### ISSUE THREAD

**Issue by discover-mohit** [google/shaka-player#3613]
**Have you read the [Tutorials](http://shaka-player-demo.appspot.com/docs/api/tutorial-welcome.html)?**
Yes

**Have you read the [FAQ](https://bit.ly/ShakaFAQ) and checked for duplicate open issues?**
Yes

**What version of Shaka Player are you using?**
3.2.0

**Please ask your question**
I am missing timelineregionenter event for few ads in a adbreak this is very random and I am unable to find a solution on the web. Can there be issue on the player side. And if not then what might be the possible reasons.

**Comment by joeyparrish** [comment]
Yes, there could be an issue on the player side.  We can never guarantee that there are no bugs in any product.  Can you please file a new issue using the bug report template so that we can get enough information to reproduce and try to solve your issue?  https://github.com/google/shaka-player/issues/new/choose

Thanks!
